coreapi/models.py

```diff
@@ -0,0 +1,85 @@
+''' creates a db handler class to do crud operations on the API '''
+
+
+class Question():
+    ''' Question class '''
+    idCounter = 0
+
+    def __init__(self, title=None, body=None):
```

Comment:
@bozicschucky why are you initializing question with none attributes? This means a question can be created with empty values.